src/web_interface/templates/base.html

```diff
@@ -49,62 +49,75 @@
     {% endblock %}
 
     <style>
-            a:link{
-                color:#2b669a;
-                text-decoration:none;
-            }
-            .clickable {
-                cursor:pointer;
-            }
-            .bg-fact {
-                  background-color: #255e54 !important;
-            }
-            .navbar-light .navbar-brand, .navbar-light .navbar-nav > li > a {
-                color: #ddd;
-            }
-            .navbar-light .navbar-brand:hover, .navbar-light .navbar-nav > li > a:hover {
-                color: #fff;
-            }
+        a:link{
```

Comment:
Beurk. Changed indentation

Reply:
more like "fixed indentation": It was eight spaces here for no apparent reason